include/SFML/Window/Mouse.hpp

```diff
@@ -172,6 +172,8 @@ class SFML_WINDOW_API Mouse
 /// sf::Mouse::setPosition(sf::Vector2i(100, 200), window);
 /// \endcode
 ///
+/// \warning sf::Mouse::XButton1 and sf::Mouse::XButton2 are not available on Linux.
+///
```

Comment:
As pointed out by @kimci86, these buttons are available through events, so we might want to move this warning to `isButtonPressed` instead